api/services/namespace.go

```diff
@@ -109,6 +109,8 @@ func (s *service) CreateNamespace(ctx context.Context, namespace request.Namespa
 		return nil, NewErrNamespaceCreateStore(err)
 	}
 
+	s.store.UserUpdateNamespacesCounter(ctx, userID, 1) // nolint: errcheck
```

Comment:
Must run in a transaction, no?

Reply:
Yes! You're right!

I don't even need these store methods; I just need to put it together with namespace's creation method.

Thank you.